ietf/nomcom/utils.py

```diff
@@ -183,7 +183,7 @@ def retrieve_nomcom_private_key(request, year):
             settings.OPENSSL_COMMAND,
             command_line_safe_secret(settings.NOMCOM_APP_SECRET)
         ),
-        private_key
+        private_key.encode("utf8")
```

Comment:
I don't think this is going to work. The nomcom private key is part of a keypair that is generated externally, and the byte sequence is not guaranteed to be valid utf8. Maybe uuencoding first would work around the issue?

Please also check that the instructions (and explanations) at  https://github.com/ietf-tools/datatracker/blob/main/ietf/templates/nomcom/chair_help.html#L9-L76 are still accurate.

Reply:
Sorry - I hadn't seen the exchange with Rich

Reply:
This should not change anything with respect to the instructions or explanations. The change here is only in handling the data that comes back from gpg. As long as the reverse operation is done when feeding it back to gpg to decrypt, it should be transparent.

I will see about switching it for a more robust handling of arbitrary bytes, otherwise someone somewhere will change a locale and we'll get a weird bug.